ci/buildspec.yml

```diff
@@ -0,0 +1,13 @@
+version: 0.2
+phases:
+  install:
+    runtime-versions:
+      nodejs: 14
+    commands:
+  build:
+    commands:
+      - npm install aws-cdk@1.104.0 -g
+      - npm install
+      - cdk deploy --verbose --all --require-approval never
```

Comment:
We need a cdk.json file populated with our values for tests, e.g. for route53 domain name, IAM users/roles for teams. 
Can we create a file under the ci (or test) to copy over to the build job?

Reply:
We absolutely can if we need to. The entire git repo gets cloned in the build container. We can also pass values as environment variables and provide them as CLI parameters to the `cdk deploy`. If there are specific context variables we need to pass please let me know.

Reply:
I am thinking context variables such as route 53 domain, domain account, subzone, IAM users (if we test teams). The reason why I brought it up, ideally if we have a file under CI folder which is a place holder for such values, we will be decoupled from the quickstart team for any changes in the future. It could be just a matter of creating a place holder file and passing it over to the CDK deploy command.

Reply:
Added as s3 bucket location containing path to a context.json. The bucket location is provided as context itself.